Lib/test/test_ipaddress.py

```diff
@@ -2277,6 +2277,39 @@ def testReservedIpv4(self):
         self.assertEqual(False, ipaddress.ip_address('128.0.0.0').is_loopback)
         self.assertEqual(True, ipaddress.ip_network('0.0.0.0').is_unspecified)
 
+    def testPrivateNetworks(self):
+        self.assertEqual(False, ipaddress.ip_network("0.0.0.0/0").is_private)
```

Comment:
FYI I would suggest using assertTrue and assertFalse for these, but it isn't a big deal, the code in this file is old and predates those existing so this at least matches surrounding code.  someone else can modernize it sometime if they wish.